input/postgres/databases.go

```diff
@@ -8,60 +8,75 @@ import (
 	"github.com/pganalyze/collector/util"
 )
 
-const databasesSQLDefaultOptionalFields = "1"
-const databasesSQLpg93OptionalFields = "datminmxid"
+const databasesSQLDefaultMxidFields = "1"
+const databasesSQLpg93MxidFields = "d.datminmxid"
+const databasesSQLDefaultMxidAgeFields = "0"
+const databasesSQLpg93MxidAgeFields = "CASE WHEN d.datminmxid <> '0' THEN mxid_age(d.datminmxid) ELSE 0 END"
```

Comment:
[note for reviewer]
I kept the "age" as 0 when
1. the data is not retrievable (before pg 9.3)
2. min xid or mxid is 0

I've seen the pattern that when min (frozen) xid is zero, the age is becoming some big number. This value being zero actually has a different meaning ([doc of pg_class](https://www.postgresql.org/docs/current/catalog-pg-class.html)):

> relfrozenxid: Zero (InvalidTransactionId) if the relation is not a table.

Technically, `datfrozenxid` is the minimum of the per-table `relminmxid` values, so it's less likely gonna be `0` (totally possible with `pg_class` side, but just not `pg_database` side), but adding that to be consistent with the relation side.